dingo/src/main/java/io/theholygrail/dingo/platform/PlatformBridge.java

```diff
@@ -36,11 +36,13 @@
     private JSWebView mWebView;
     private Handler mHandler;
     private JsonTransformer mJsonTransformer;
+    private PlatformBridgeCallback mCallback;
 
-    public PlatformBridge(Context context, JSWebView webView, JsonTransformer transformer) {
+    public PlatformBridge(Context context, JSWebView webView, JsonTransformer transformer, PlatformBridgeCallback callback) {
```

Comment:
Maybe a NonNull annotation on the callback here since we use it without nullchecks later.